### PR TITLE
updated PR Template

### PR DESCRIPTION
**Pull request by DerMolly**
changed PR Template to indicate DCO and change handling of unnecessary checkboxes.

### CODE REVIEW

**Review by InnayTool — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -6,12 +6,13 @@ This PR fixes/adds/improves/...
...
 ### Steps
 
-<!-- please tick steps this PR performs (if something is not necessary, please tick anyway to indicate you considered it) -->
+<!--please tick steps this PR performs (if something is not necessary, please remove it) -->
 
 - [ ] added implementation
 - [ ] added / updated tests
 - [ ] added / updated documentation
 - [ ] extended changelog
+- [ ] I read the `CONTRIBUTING.md` and signed-off my commits to accept the DCO.
```
Comment (davidmehren) [inline .github/pull_request_template.md:15]:
```suggestion
- [ ] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/master/CONTRIBUTING.md) and signed-off my commits to accept the DCO.
```
Maybe link directly to the docs?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -6,12 +6,13 @@ This PR fixes/adds/improves/...
 
 ### Steps
 
-<!-- please tick steps this PR performs (if something is not necessary, please tick anyway to indicate you considered it) -->
+<!--please tick steps this PR performs (if something is not necessary, please remove it) -->
```
Comment (davidmehren) [inline .github/pull_request_template.md:9]:
```suggestion
<!-- Please tick all steps this PR performs (if something is not necessary, please remove it) -->
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -6,12 +6,13 @@ This PR fixes/adds/improves/...
...
 
 - [ ] added implementation
```
Comment (davidmehren) [inline .github/pull_request_template.md:11]:
```suggestion
- [ ] Added implementation
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -6,12 +6,13 @@ This PR fixes/adds/improves/...
...
 - [ ] added / updated tests
```
Comment (davidmehren) [inline .github/pull_request_template.md:12]:
```suggestion
- [ ] Added / updated tests
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -6,12 +6,13 @@ This PR fixes/adds/improves/...
...
 - [ ] added / updated documentation
```
Comment (davidmehren) [inline .github/pull_request_template.md:13]:
```suggestion
- [ ] Added / updated documentation
```

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -6,12 +6,13 @@ This PR fixes/adds/improves/...
...
 - [ ] extended changelog
```
Comment (davidmehren) [inline .github/pull_request_template.md:14]: We don't really have a "PRs extend the changelog" policy here, maybe remove this step?

**Review by davidmehren — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (DerMolly) [reply .github/pull_request_template.md:15]: Should we link to the DCO then too?

File: .github/pull_request_template.md (hunk 6 above)
Comment (DerMolly) [reply .github/pull_request_template.md]: fixed

File: .github/pull_request_template.md (hunk 1 above)
Comment (davidmehren) [reply .github/pull_request_template.md:14]: We could, but the contribution docs one is supposed to read already do :D

Comment (DerMolly) [reply .github/pull_request_template.md]: fixed

**Review by davidmehren — APPROVED** [review]

**Review by ErikMichelson — APPROVED** [review]

**Pull request merged**